tools/clang/test/DXC/root_sig.test

```diff
@@ -26,8 +26,9 @@
 
 // Set root signature when already has one should succeed
 // RUN: %dxc %t.define.cso /dumpbin /setrootsignature %t.rootsig.cso /Fo %t.smoke.rsadded.cso
-// Make sure %t.smoke.rsadded.cso exist.
-// RUN: ls %t.smoke.rsadded.cso
+// Make sure %t.smoke.rsadded.cso not empty.
+// RUN:FileCheck --input-file=%t.define.cso %s --check-prefix=CSO
+// CSO:{{.+}}
```

Comment:
This is probably fine for this change, but we should have a tool for inspecting the contents of the `cso` file so that we can ensure it is generated correctly.